awq/utils/exllama_utils.py

```diff
@@ -0,0 +1,80 @@
+import torch
+
+
+AWQ_ORDER = [0, 2, 4, 6, 1, 3, 5, 7]
+AWQ_REVERSE_ORDER = [0, 4, 1, 5, 2, 6, 3, 7]
```

Comment:
How is this reverse order derived from the packing order?

Reply:
Noticing that AWQ_ORDER is a permutation, for every i in [0,..,7] `AWQ_REVERSE_ORDER[AWQ_ORDER[i]] = i`
Or in other words, if `AWQ_ORDER` takes element in index 2 and puts it at index 4 (`AWQ_ORDER[2]=4`), `AWQ_REVERSE_ORDER` should take the element in index 4 and put it at index 2 (`AWQ_REVERSE_ORDER[4]=2`).